Assets/Scripts/PlayerController.cs

```diff
@@ -311,4 +301,52 @@ private void move()
     rotate();
     animateMove();
   }
+
+  private void UseObjectInteraction(string interactionName)
+  {
+    InteractionType interactionType = InteractionType.UseMouth;
+
+    switch (interactionName)
+    {
+      case "UsePaw":
+        interactionType = InteractionType.UsePaw;
+        break;
+      case "UseBody":
+        interactionType = InteractionType.UseBody;
+        break;
+      default:
+        break;
+    }
+
+    interactable.HandleInteraction(interactionType);
```

Comment:
this is where those animations return. we can use them to select the appropriate animation to play here.

we're also passing in the `InteractionType` to allow the `Interactable` to do whatever type specific behavior it may need to do.